src/main/java/com/zontik/groshiky/dao/IUserDao.java

```diff
@@ -8,4 +8,5 @@
 @Repository
 public interface IUserDao extends JpaRepository<User, Integer> {
     User findByLogin(String login);
+
```

Comment:
The same as before... Do not create not necessary changes, please. It makes bigger PR for no reason.